tests/failpoints/test_engine.rs

```diff
@@ -611,6 +611,7 @@ fn test_recycle_with_stale_logbatch_at_tail() {
         target_file_size: ReadableSize::kb(2),
         purge_threshold: ReadableSize::kb(4),
         enable_log_recycle: true,
+        format_version: Version::V1,
```

Comment:
Why?

Reply:
Here, default Version has been updated to V2 to enable log recycling defaultly, this case should be compatible to this modification.

The original design of this case is to mock a case that RaftEngine does recycling on `V1` log files, and it cannot recognize the recycled part in the recycled file, as `V1` logs has no signatures.